### PR TITLE
Update simulator config

### PR DESCRIPTION
**Pull request by citlaligm**
This PR merges PR #85 from @devinaconley and the updated PR #84.

Removes all the ego configuration from the simulator config since now the information for color and the model of the vehicle is contained on the replay files.
```
 "ego_config":{
    "body":{
      "type": "Blueprint'/Game/Vehicles/crossover_monoDrive_01.crossover_monoDrive_01_C'",
      "color": "CarPaint_White.CarPaint_White"
    }
  }
```
I ran all the cpp examples on windows, none seem to complaint.

### CODE REVIEW

File: examples/cpp/dev/all_cameras.cpp [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    // c_config.listen_port = 8101;
+    // sensors.push_back(std::make_shared<Sensor>(std::make_unique<CameraConfig>(c_config)));
+
+    // CameraConfig gc_config;
+    // gc_config.server_ip = sim0.getServerIp();
+    // gc_config.server_port = sim0.getServerPort();
+    // gc_config.resolution = Resolution(512,512);
+    // gc_config.listen_port = 8102;
+    // gc_config.channels = "gray";
+    // sensors.push_back(std::make_shared<Sensor>(std::make_unique<CameraConfig>(gc_config)));
```
Comment (bradleynull) [inline examples/cpp/dev/all_cameras.cpp:68]: Uncomment the above code?

File: examples/cpp/dev/all_cameras.cpp [hunk 2]
```diff
@@ -0,0 +1,100 @@
+#include <iostream>		  // std::cout
```
Comment (bradleynull) [inline examples/cpp/dev/all_cameras.cpp:1]: This example is not in the `CMakeLists.txt` and is not being built.

File: monodrive/core/src/sensor_config.h [hunk 3]
```diff
@@ -270,10 +270,10 @@ class DepthCameraConfig : public CameraConfig
     }
 };
 
-class OccupancyGridConfig : public CameraConfig
+class OccupancyGridConfig : public SemanticCameraConfig
```
Comment (bradleynull) [inline monodrive/core/src/sensor_config.h:273]: The occupancy grid sensor is now causing a crash on the server side: 
image

This was found when running the occupancy grid dev example.

Call stack:

``` 	
KernelBase.dll!00007ffeb6c73e49()	Unknown
 	vcruntime140.dll!00007ffea09d4411()	Unknown
 	ntdll.dll!00007ffeb93304a6()	Unknown
 	opencv_world410.dll!00007ffe1d6dadae()	Unknown
 	opencv_world410.dll!00007ffe1d5df643()	Unknown
 	opencv_world410.dll!00007ffe1d71d159()	Unknown
 	opencv_world410.dll!00007ffe1d863f6b()	Unknown
 	opencv_world410.dll!00007ffe1d86ddce()	Unknown
 	opencv_world410.dll!00007ffe1d841034()	Unknown
>	UE4Editor-OpenCVLibrary.dll!OpenCVInterface::cvtColor(cv::Mat * src, cv::Mat * dst, int type) Line 33	C++
 	UE4Editor-monoDriveCameraSensor.dll!UCameraSensorComponent::PostProcessImage(void * ImageBuffer, int Width) Line 250	C++
 	UE4Editor-monoDriveCameraSensor.dll!UCameraSensorComponent::ResolveFrameData::__l2::<lambda>(FRHICommandListImmediate & RHICmdList) Line 399	C++
 	[Inline Frame] UE4Editor-monoDriveCameraSensor.dll!TEnqueueUniqueRenderCommandType<`UCameraSensorComponent::ResolveFrameData'::`2'::ResolveRenderTargetName,void <lambda>(FRHICommandListImmediate &)>::DoTask(ENamedThreads::Type) Line 198	C++
 	UE4Editor-monoDriveCameraSensor.dll!TGraphTask<TEnqueueUniqueRenderCommandType<`UCameraSensorComponent::ResolveFrameData'::`2'::ResolveRenderTargetName,void <lambda>(FRHICommandListImmediate &)>>::ExecuteTask(TArray<FBaseGraphTask *,TSizedDefaultAllocator<32>> & NewTasks, ENamedThreads::Type CurrentThread) Line 847	C++
 	[External Code]	
```

**Comment by citlaligm** [comment]
@bradleynull the branch needed an update from development. 
This branch meant to update only the `simulator.json` files. For the file you mentioned, I think doesn't need any change because it does not have the ego config tag on it. The crash you mention was fixed yesterday.

**Pull request merged**